frontend/src/pages/pipelines/global/modelCustomization/FineTunePage.tsx

```diff
@@ -48,18 +49,21 @@ const FineTunePage: React.FC<FineTunePageProps> = ({
   const { project } = usePipelinesAPI();
   const { state }: { state?: ModelCustomizationRouterState } = useLocation();
   const { hyperparameters } = filterHyperparameters(ilabPipelineVersion);
-  const { getAllValidationIssues } = React.useContext(ValidationContext);
-  const hasValidationErrors =
-    Object.keys(getAllValidationIssues(['inputPipelineParameters'])).length > 0;
   const [connectionTypes] = useWatchConnectionTypes();
   const ociConnectionType = React.useMemo(
     () => connectionTypes.find((c) => c.metadata.name === 'oci-v1'),
     [connectionTypes],
   );
 
+  const invalidParameters = React.useMemo(() => {
+    const parameters = getInputDefinitionParams(ilabPipelineVersion);
+    const result = pipelineParameterSchema.safeParse(parameters ?? {});
+    return result.error?.issues ?? [];
+  }, [ilabPipelineVersion]);
+
```

Comment:
I think we need to have this in `ModelCustomizationForm.tsx` to filter out the sidebar section

Currently we have this in `ModelCustomizationForm.tsx`, which is invalid now
  ```
const pipelineParameterErrors =
    Object.keys(formValidation.getAllValidationIssues(['inputPipelineParameters'])).length > 0;
  const filteredFineTunePageSections = React.useMemo(
    () =>
      pipelineParameterErrors
        ? Object.values(FineTunePageSections).filter(
            (section) =>
              section === FineTunePageSections.PROJECT_DETAILS ||
              section === FineTunePageSections.PIPELINE_DETAILS,
          )
        : FineTunePageSections,
    [pipelineParameterErrors],
  );

```

Reply:
i will just move it up